.github/bin/dist-npm.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Submit npm packages to npmjs.org.
+# Note that this script is intended to be run by GitHub Actions.
+set -e
+
+# shellcheck source=constants.sh
+. "$(dirname "$0")/constants.sh"
+
+if [ "$NODE_AUTH_TOKEN" = "" ]; then
+  echo "This script requires NODE_AUTH_TOKEN envrionment variable." \
+    > /dev/stderr
+  exit 1
+fi
+
+# Publish a package only if the repository is upstream (planetarium/libplanet)
+# and the branch is for releases (master or *-maintenance or 9c-*).
+# shellcheck disable=SC2235
+if [ "$GITHUB_REPOSITORY" = "planetarium/libplanet" ] && [[ \
```

Comment:
this check seems duplicated with [dist-nuget.sh's](https://github.com/planetarium/libplanet/blob/b5dcfbe/.github/bin/dist-nuget.sh#L25-L29)... Can we reuse it?

Reply:
🙄 Yeah, I did copy-and-pasted this piece from the *dist-nuget.sh* script… I am going to merge two check conditionals to a common script (probably *constants.sh*?).